src/app/Home/components/HomeComponent.js

```diff
@@ -1,14 +1,71 @@
 /* eslint-disable react/destructuring-assignment */
 import React, { Component } from 'react';
-import TestSaga from './testSaga';
+import Modal from './onBoardingModal';
+import LoginModal from './logInPointsModal';
 
 // eslint-disable-next-line react/prefer-stateless-function
 export class HomeComponent extends Component {
-  render() {
-    return (
-      <TestSaga />
-    );
+  constructor() {
+    super();
+
+    this.state = {
+      isShowing: false,
+      isShowingLoginPoints: false,
+    };
+  }
+
+  componentDidMount() {
+    this.setState({
+      isShowing: true,
+    });
   }
+
+closeModalHandler = () => {
+  this.setState({
+    isShowing: false,
+  });
+}
+
+openModalLoginPointsHandler = () => {
+  this.setState({
+    isShowingLoginPoints: true,
+  });
+}
+
+closeModalLoginPointsHandler = () => {
+  this.setState({
+    isShowingLoginPoints: false,
+  });
+}
+
+
+render() {
+  return (
+    <div>
+      { this.state.isShowing
+        ? <div role='presentation' onClick={this.closeModalHandler} className='back-drop' /> : null }
+      <h1 className='home__title'>Dashboard</h1>
+      <button type='button' onClick={this.openModalLoginPointsHandler}>Log-in points</button>
+      {
+        !this.state.isShowingLoginPoints
```

Comment:
if you are going to use something in state in more than one place you can destructure so that you don't have to use this.state.blablabla, it saves time and cleaner code.